src/main/java/qupath/ext/bioimageio/BioimageIoCommand.java

```diff
@@ -102,7 +107,8 @@ public class BioimageIoCommand {
 	public void promptForModel() {
 		
 		// TODO: In the future consider handling .zip files
-		var file = Dialogs.promptForFile(title, null, "BioImage Model Zoo spec", "", ".yml", ".yaml");
+		var file = FileChoosers.promptForFile(title,
+				new FileChooser.ExtensionFilter("BioImage Model Zoo YAML file", "*.yml", "*.yaml"));
```

Comment:
You can use [`FileChoosers.createExtensionFilter(String description, String…exts)`](https://github.com/qupath/qupath-fxtras/blob/f4ca3eb1697e9f9c6cf1962da35471f0de1ae93c/src/main/java/qupath/fx/dialogs/FileChoosers.java#L336).

Doesn’t save much, admittedly, but a goal of `FileChoosers` was that you shouldn’t have to import `FileChooser` or know about its API as well.

Reply:
Neat, although not the worst API anyways